### PR TITLE
Hh/dtm 511/implement basic secret handling

### PR DESCRIPTION
**Pull request by hal-henningsmoen**
✅ Review ✅ Merge
@dhallman this can be merged independent of deploying the new secrets functionality of Hodor.  
Added examples of secrets configuration related files.

### CODE REVIEW

**Comment by hal-henningsmoen** [comment]
Good point.  Will do.

**Comment by hal-henningsmoen** [comment]
This is ready to merge.  @dhallman

File: config/private_example_folder/secrets.edn [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{:production { :db1.username bill_over
```
Comment (russellwhitaker) [inline config/private_example_folder/secrets.edn]: Clojure/EDN style nitpick: remove spaces in braces.

Comment (hal-henningsmoen) [inline config/private_example_folder/secrets.edn]: Thanks @russellwhitaker.  Done.

**Comment by rdhallman** [comment]
LGTM. Merging.

**Pull request merged**